### PR TITLE
[TOSCA YAML] CSAR Import

### PR DESCRIPTION
**Pull request by v-yussupov**
This PR adds import of CSARs in YAML format into Winery.
In this version of import, only the artifacts for node types and node templates are processed.

- [x] Ensure that you followed http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request.
- [x] Branch name checked. That means, the branch name starts with `thesis/`, `fix/`, ...
- [x] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Ensure to use auto format in **all** files
- [ ] Ensure that you appear in `NOTICE` at Copyright Holders
- [ ] Tests created for changes
- [ ] Documentation updated (if needed)
- [ ] Screenshots added (for UI changes)

### CODE REVIEW

File: org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/converter/Y2XConverter.java [hunk 1]
```diff
@@ -991,21 +1003,33 @@ private ParameterDefinition convert(TParameterDefinition node, String name) {
...
+            importType = Namespaces.TOSCA_NS;
+        } else {
+            importType = name;
+        }
+
+        TImport.Builder builder = new TImport.Builder(importType)
+            .setNamespace(node.getNamespaceUri())
+            .setLocation(node.getFile());
+        
+        /*Reader reader = Reader.getReader();
```
Comment (miwurster) [inline org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/converter/Y2XConverter.java:1017]: Remove if this is no longer required.

File: org.eclipse.winery.model.csar.toscametafile/src/main/java/org/eclipse/winery/model/csar/toscametafile/TOSCAMetaFileAttributes.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
     final public static String TOSCA_META_VERSION_VALUE = "1.0";
     final public static String CSAR_VERSION = "CSAR-Version";
     final public static String CSAR_VERSION_VALUE = "1.0";
+    final public static String YAML_CSAR_VERSION_VALUE = "1.1";
```
Comment (miwurster) [inline org.eclipse.winery.model.csar.toscametafile/src/main/java/org/eclipse/winery/model/csar/toscametafile/TOSCAMetaFileAttributes.java:26]: Should we use `1.3`?! ... or is this another version?

Comment (v-yussupov) [reply org.eclipse.winery.model.csar.toscametafile/src/main/java/org/eclipse/winery/model/csar/toscametafile/TOSCAMetaFileAttributes.java:26]: This describes the Metafile version, taken from the spec:
https://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.3/os/TOSCA-Simple-Profile-YAML-v1.3-os.html#_Toc454457870

File: org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/converter/Y2XConverter.java (hunk 1 above)
Comment (v-yussupov) [reply org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/converter/Y2XConverter.java]: done

**Pull request merged**